kythe/web/site/install-macos.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: page
+title: Getting Started on macOS
+permalink: /getting-started-macos/
+---
+
+* toc
+{:toc}
+
+This document extends the [Getting Started guide](getting-started.md) to cover
+installation of external dependencies on a macOS system. You will still need to
+follow all the other Getting Started instructions to get a working Kythe build.
+
+
+## Installing Developer Tools
+
+**Xcode**: To build on macOS, you will need Xcode installed. You can do this
```

Comment:
Did you need to run xcode-select? Or accept a license from Apple? In the past you had to explicitly accept a license otherwise things would cryptically fail.

Reply:
It appears the Homebrew installer will run it for you. I ran it expecting the result you describe, but it just said "you've already installed them, go away".

Reply:
nice